build/release/README.md

```diff
@@ -0,0 +1,60 @@
+# lnd's Reproducible Build System
+
+This package contains the container that the `lnd` project uses in order to
+build binaries for each new release. As of Go 1.13, with some new build flags,
+binaries are now reproducible, allowing developers to build the binary on
+distinct machines, and end up with a byte-for-byte identical binary.  However,
+this wasn't _fully_ solved in Go 1.13, as the build system still includes the
+directory the binary is built into the binary itself. As a result, our scripts
+utilize a work around needed until Go 1.13.2.  
+
+## Building a New Release
+
+### MacOS/Windows
+
+The first requirement is to have [`docker`](https://www.docker.com/)
+installed locally and running.
+
+Next, change into the build/release directory and run the following command:
+```
+docker build -t=lnd-release .
+```
+
+Once that command has finished executing, our build environment will now be
+ready.
+
+At this point, we're now ready to build the latest release. The way the
+container is set up, we'll mount the present `lnd` directory _into_ the
+container at `/tmp/build/lnd`. This way we can use the existing code checked
+in, but ensure the build will be deterministic from the container's
+perspective.  After the container build is done, a new file should be
+available in the top-level `lnd` directory, which will include the final
+release directory.
+
+To build a release, run the following command (from 'build/release`):
+```
+docker run -it -v $(pwd)/../..:/tmp/build/lnd lnd-release <TAG>
+```
+
+Where `<TAG>` is the name of the next release of `lnd`.
+
+### Linux
```

Comment:
```suggestion
### Linux/macOS/Windows (WSL)
```

Reply:
I would also move this section above the docker container since most users will fall under this category.